samples/matter/common/src/bridge/Kconfig

```diff
@@ -76,4 +78,8 @@ config BRIDGE_BT_CONNECTION_LATENCY
 	int "It allows the peripheral to skip waking up for a certain number of connection events if it does not have any data to send"
 	default 0
 
+config BRIDGE_FORCE_BT_CONNECTION_PARAMS
+	bool "Determines whether the Matter bridge forces connection parameters or accepts the Bluetooth LE peripheral device selection"
+	default n
```

Comment:
Wasn't this enabled by default before?

Reply:
Oh yes, it was, sorry i misunderstood this comment
https://github.com/nrfconnect/sdk-nrf/pull/17128#discussion_r1743181975

Reply:
Ah ok, I see. I think it would be good to have it enabled by default and mention in the docs that it can be disabled by setting kconfig to n.